README.md

```diff
@@ -71,6 +71,7 @@ OPTIONS:
                                      Exits with a non-zero status code if there were any failures.
         --report-timeout N           Fail if build is not finished after N seconds. Only applicable if --report is enabled (default: 3600).
         --max-requeues N             Retry failed examples up to N times before considering them legit failures (default: 3).
+        --fail-fast N                Stop running the suite after N failed tests (default: nil).
```

Comment:
For consistency with the other messages (e.g. `--report`), consider changing this to:

> Abort build with a non-zero status code after N failed examples.

The `(default: nil)` part isn't needed since by defaults this is disabled (so it's implied).